py-phylo2vec/src/lib.rs

```diff
@@ -30,6 +31,26 @@ fn build_newick(input_ancestry: Vec<(usize, usize, usize)>) -> String {
     newick_string
 }
 
+#[pyfunction]
+#[pyo3(signature = (n_leaves, ordered=false, /), text_signature = "(n_leaves, ordered=False, /)")]
```

Comment:
@lsetiawan One interesting point is that the default parameter used by the python binding is actually set on this line (not the Rust default we have set). Meaning we could remove the SampleOrdering enum and leave it as bool.

Reply:
Awesome! Great to hear you're finding out about these things 😄